### PR TITLE
Publication Episode 

### PR DESCRIPTION
**Pull request by LydiaFrance**
<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this contribution is for instructor training, please email the link to this contribution to
checkout@carpentries.org so we can record your progress. You've completed your contribution
step for instructor checkout by submitting this contribution!

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution. Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact The Carpentries Team at team@carpentries.org.

You may delete these instructions from your comment.

\- The Carpentries
</details>

### CODE REVIEW

File: _episodes/10-publication.md [hunk 1]
```diff
@@ -12,6 +12,136 @@ keypoints:
...
+<img src="https://the-turing-way.netlify.app/_images/evolution-open-research.jpg" alt="drawing" width="500"/> 
+
+While the output of research projects is usually centred around publishing a journal article, this format of science communication and knowledge sharing is increasingly restrictive with the new ways scientific research is conducted. The requirements from journals themselves is also expanding, you are now often asked to upload data sets and code as part of your publication. Releasing data is increasingly a requirement from funding bodies, and outputs from research groups can go beyond a single paper, releasing tools and methods that can be used worldwide.
+
+In general there are different degrees of openness. 
+
+- Fully private data and code, unavailable
+- Pseudo-open -- "available on request"
+- Released static code parallel to a research paper, see Zenodo or Figshare
+- Open online repository -- CRAN, GitHub
```
Comment (EKaroune) [inline _episodes/10-publication.md:26]: I don't think we want call Github a repository - maybe call it an open-source project tool

If this section is about publication, we need to be focused on long-term archiving so stress Trusted Repositories such as zenodo, osf, figshare, etc.

File: _episodes/10-publication.md [hunk 2]
```diff
@@ -12,6 +12,136 @@ keypoints:
...
+<img src="https://i.imgur.com/I9EKxdF.png" alt="drawing" width="800"/> 
+
+There can be specific requirements to keep code bases and/or data private. See the section below for good and not so good reasons for keeping work private.
+
+You can release code and data associated with a research article as a series of files/folders. If your project follows the folder template introduced in a previous episode, for example:
+<img src="https://github.com/jcolomb/gintonic-blog/raw/forreview/50_figures/pulication_ready_figures/2draft-templates.png" alt="drawing" width="300"/> 
+
+> Examples of a template folder tree for a computational project. https://github.com/tonic-team/Tonic-Research-Project-Template
+
+You could bundle folders into a `.zip` file and upload it to Zenodo.
```
Comment (EKaroune) [inline _episodes/10-publication.md:62]: Maybe add in - You can also link your github account to zenodo, which allows you to push releases of different versions of your repository to Zenodo for long term storage/archiving and to get a digital object identifier (DOI).

File: _episodes/10-publication.md [hunk 3]
```diff
@@ -12,6 +12,136 @@ keypoints:
...
+>
+> - **Safe** — your research is stored safely for the future in CERN’s Data Centre for as long as CERN exists.
+> - **Trusted** — built and operated by CERN and OpenAIRE to ensure that everyone can join in Open Science.
+> - **Citeable** — every upload is assigned a Digital Object Identifier (DOI), to make them citable and trackable.
+No waiting time — Uploads are made available online as soon as you hit publish, and your DOI is registered within seconds.
+> - **Open or closed** — Share e.g. anonymized clinical trial data with only medical professionals via our restricted access mode.
+> - **Versioning** — Easily update your dataset with our versioning feature.
+> - **GitHub integration** — Easily preserve your GitHub repository in Zenodo.
+> - **Usage statistics** — All uploads display standards compliant usage statistics
+
```
Comment (EKaroune) [inline _episodes/10-publication.md:80]: Do we want to add in instructions about how to link github and zenodo? or maybe a link? 

This link looks good - https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content

File: _episodes/10-publication.md (hunk 1 above)
Comment (jcolomb) [reply _episodes/10-publication.md:26]: the question is still open I think, as for recommendation for FAIR software often starts with having it on a git repo system, only mentioning archives later on. I think there is no hierarchy of openes between the two. The important missing parterre  is the license.

**Comment by jcolomb** [comment]
to be continued during the next sprint, merging first as explained in #52

**Pull request merged**